scss/_variables.scss

```diff
@@ -1180,15 +1180,28 @@ $form-check-btn-check-disabled-opacity:    null !default;
 $form-check-inline-margin-end:    1rem !default;
 
 // Boosted mod: Star rating
-$form-star-size:                  1.5625rem !default;
-$form-star-size-sm:               1.25rem !default;
-$form-star-margin-between:        -.125rem !default;
+$form-star-size:                        1.5625rem !default;
+$form-star-size-sm:                     1.25rem !default;
+$form-star-margin-between:              -.125rem !default;
+
+$form-star-rating-checked-color:        var(--#{$prefix}link-hover-color) !default;
```

Comment:
The following 3 Sass variables use unrelated custom properties. There's no logical link between:
* star rating checked color and link hover color
* star rating unchecked color and code color
* star rating hover color and emphasis color

Maybe it was needed with the dark mode version but we are not handling it right now